apps/src/StudioApp.js

```diff
@@ -391,7 +409,14 @@ StudioApp.prototype.init = function(config) {
     }
     var bubble = document.getElementById('bubble');
     dom.addClickTouchEvent(bubble, _.bind(function() {
-      this.showInstructions_(config.level, false);
+      var hintsToShow = this.authoredHints_.filter(function(hint){
+        return hint.alreadySeen === false;
+      });
```

Comment:
I'm mostly just looking forward to not having to type `prototype` dozens of times in every file